public/sitemap.xml

```diff
@@ -0,0 +1,64 @@
+<!-- public/sitemap.xml -->
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://findrecruitandfollowup.nihr.ac.uk</loc>
+    <lastmod></lastmod>
```

Comment:
This `lastmod` tag should not be left blank, important for SEO.  Complete in this format:

YYYY-MM-DD

Reply:
Added